### ISSUE TITLE
Service crash on wireguard connection

### ISSUE THREAD

**Issue by averrin** [corrad1nho/qomui#28]
1. Install qomui from AUR
2. Import wireguard config from streisand
3. Try to connect

Result:
qomui-service output
```
net.ipv6.conf.all.disable_ipv6 = 0
Error: argument "bypass_qomui" is wrong: invalid table ID

Error: argument "bypass_qomui" is wrong: invalid table ID

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/qomui/tunnel.py", line 42, in run
    self.wireguard()
  File "/usr/lib/python3.7/site-packages/qomui/tunnel.py", line 61, in wireguard
    shutil.copyfile("{}/{}".format(ROOTDIR, self.server_dict["path"]), path)
NameError: name 'shutil' is not defined
[1]    12006 abort      sudo qomui-service
```

**Comment by corrad1nho** [comment]
Sorry for the late answer. I can confirm this issue, it's caused by a missing import statement in tunnel.py that affects manual WireGuard config imports. This will be fixed in the next release. In the meantime, you can either downgrade to version 0.7.3 to avoid the issue or add "import shutil" to the imports in /usr/lib/python3.7/site-packages/qomui/tunnel.py manually.

**Comment by corrad1nho** [comment]
Has been fixed in v0.8.0. Sorry for taking so long.
